force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -214,8 +263,14 @@ public with sharing class ClassesObjectsMethods {
     public static Integer calculateYearDifference(Date d1, Date d2) {
 
         // Subtract the difference in years
+        Integer result = 0;
 
-        return null; // Replace null with the variable you used to store the result
+        if (d1 != null && d2 != null){
+            result = d2.year() - d1.year();
```

Comment:
Try to use the abs() method to get a positive number every time. Check the return statement on line 261